static/js/message_edit.js

```diff
@@ -823,6 +823,27 @@ export function save_message_row_edit(row) {
     // The message will automatically get replaced via message_list.update_message.
 }
 
+export function delete_message_row_edit(row) {
+    // This function is used to remove the content of the message by editing it.
+
+    const message_id = rows.id(row);
+    const message = current_msg_list.get(message_id);
+
+    row.find(".message_edit_content").val("");
+    const new_content = row.find(".message_edit_content").val();
+
+    const request = {message_id: message.id};
+
+    if (new_content !== message.raw_content) {
+        request.content = new_content;
+    }
```

Comment:
Hmm, what is the purpose of this block?

Reply:
So, I had used similar logic as `save_message_row_edit` here. It is unnecessary to check if the message's content has changed and hence we only need to update the message with empty string. I will update that block and only keep `request.content = new_content;`. Thanks for the review.